documentation/getting-started/README.md

```diff
@@ -22,85 +22,83 @@ You can install TestCafe Studio on MacOS, Windows or Linux. See [Installation](.
-The assertion is added to the end of the test.
+1. Click the ![Record test actions](../../images/getting-started/record-test-icon.png) **Record test actions** button on the **Test Editor**'s toolbar to continue recording the test. TestCafe Studio replays the previous test actions and waits for you to add new actions.
 
-Next, specify the assertion parameters in the **Test Editor**:
+2. Click the ![Equals assertion](../../images/getting-started/assertion-eql-icon.svg) button in the **Actions** panel's **Assertions** section to add the assertion to the test. TestCafe Studio adds the assertion to the end of the test and allows you to specify the assertion's parameters.
 
-1. Click the button ![Pick a target element](../../images/getting-started/element-picker-icon.png) next to the **Actual** field to select an element whose property you are going to check. This displays the tested webpage where you can select the target element. Hover over the page header that says *'Thank you, ...'* to show the **Element Selector** popup that contains a CSS selector identifying the page header.
+3. Select an element whose property you are going to check. To do this, click the ![Pick a target element](../../images/getting-started/element-picker-icon.png) button next to the **Actual** field. This displays the browser with a tested webpage. Hover the mouse cursor over the page header that says *'Thank you, ...'*. The selector popup displays the [Element Selector](../guides/record-tests/test-actions/on-page-actions/action-parameters.md#element-selector) that identifies the page header.
 
     ![The Element Selector popup](../../images/getting-started/selector-popup.png)
 
-    Click the page header to add the selector to the **Actual** field. To view or select alternative selectors that identify the same element, click the ![Choose the selector](../../images/getting-started/id-bar-icon.png) arrow button next to the **Actual** field.
+    Click the page header to add the selector to the **Actual** field.
 
-2. Select the property you want to verify from the list. The **Value** field displays the current property value.
+4. Select the property you want to verify from the list.
 
     ![The Properties list](../../images/getting-started/properties-list.png)
 
-3. Enter the object property's expected value in the **Expected** field. To add the current property value, click the ![Copy the value to the Expected field](../../images/getting-started/copy-value-icon.png) button.
+    TestCafe Studio generates the element property expression and adds it to the **Actual** field. It also evaluates the specified expression and shows the result in the **Value** field.
+
+5. Specify the object property's expected value in the **Expected** field. You can click the ![Copy the value to the Expected field](../../images/getting-started/copy-value-icon.png) button to add the current property value to the field.
```

Comment:
I'm confused a bit

> element property expression

> 5. Specify the object property's

Is 'element property' the same as 'object property'?